cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -0,0 +1,15 @@
+
+#include <gtest/gtest.h>
+#include <ray/api.h>
+#include <ray/api/ray_config.h>
+
+using namespace ray::api;
+
+TEST(RayClusterModeTest, PutTest) {
+  ray::api::RayConfig::GetInstance()->run_mode = RunMode::CLUSTER;
+  Ray::Init();
+  auto obj1 = Ray::Put(12345);
+  auto i1 = obj1.Get();
```

Comment:
Should we do `Ray::Get(obj1)` instead? @raulchen @SongGuyang

Reply:
We should support both. `obj1.Get()` is more object-oriented. While `Ray::get` is needed to align with Python and support multiple objects.

Reply:
I have made a  PR. https://github.com/ray-project/ray/pull/9809